changelogs/current.yaml

```diff
@@ -76,6 +76,19 @@ minor_behavior_changes:
 - area: generic_proxy
   change: |
     Update the stats prefix of generic proxy from ``<stats_prefix>`` to ``generic_proxy.<stats_prefix>``.
+- area: circuit breakers
+  change: |
+    Previously, runtime overrides could be used to dynamically change the retry circuit breaking behavior
+    between using a static limit and using the retry budget, without changing any configuration.
+    This is no longer supported as we deprecate that interface and move to using the retry_admission_control
+    extension point instead.
+    This behavioral change can be reverted by setting ``envoy.reloadable_features.use_retry_admission_control`` to false.
+    Note that this will also mean you will be unable to use the new retry admission control extension point.
```

Comment:
Runtime flags are an Envoy-specific thing, not an xDS thing, so I'll let one of the Envoy maintainers comment on this.  From an xDS perspective, it doesn't seem problematic for runtime flag to control whether a new xDS field is used, since that's not really any different from an older client that doesn't support the new field in the first place.

Reply:
The old static limiter or retry budget should be supported forever by default (although we can moved the code to an extension implementation).

A runtime guard should be unnecessary for this patch. This patch self should ensure only when the new limiter is used, the behavior will be changed.